shiny/ui/_include_helpers.py

```diff
@@ -227,8 +231,9 @@ def get_hash(path: str, include_files: bool) -> str:
     return hash_deterministic(key)
 
 
-def get_file_key(path: str) -> str:
-    return path + "-" + str(os.path.getmtime(path))
+def get_file_key(path: Path | str) -> str:
+    path_str = str(path) if isinstance(path, Path) else path
+    return path_str + "-" + str(os.path.getmtime(path_str))
```

Comment:
If `path` is a string, `str(path)` will return the value unchanged, so you could simplify for clarity:

```suggestion
    path = str(path)
    return path + "-" + str(os.path.getmtime(path))
```

Although I personally prefer working with `Path` objects, like this:

```suggestion
    path = Path(path)
    return str(path) + "-" + str(path.stat().st_mtime)
```